sass/_blog.scss

```diff
@@ -100,7 +125,6 @@ article.post {
 
         flex: 1 1;
         max-width: 320px;
-        margin-top: 1rem;
```

Comment:
Hm this seems odd since it doesnt seem to affect the call-to-action code. Am I missing something? :)

Reply:
Oh right sorry it was a cosmetic change I did along the way when checking the CTA changes didn't break the layout on pages.

Not strictly necessary and we could split in a separate PR, but I figured it was so minor we could bring it with us along the way

Reply:
sounds reasonable to keep then :)